wasmcloud_host/lib/wasmcloud_host_web/live/page_live.html.leex

```diff
@@ -93,7 +93,7 @@
             <use xlink:href="/coreui/free.svg#cil-basket"></use>
           </svg>
         </div>
-        <div class="text-value-lg">0.50.0</div>
+        <div class="text-value-lg"><%= Mix.Project.config()[:version] %></div>
```

Comment:
`Mix.project` is unavailable from a release. Not sure if this will blow up in a release version.

Reply:
Heh I was just commenting that, I built a release to check and you're correct, it just won't show:
![image](https://user-images.githubusercontent.com/12040685/127005024-ee7208f7-4105-4c9a-8ff9-e28ea0ee893e.png)

I'll see if there's another way around, but if not, do you prefer showing hardcoded or not showing that widget?

Reply:
@autodidaddict Found a prod-friendly way using the Application module and an environment variable in `mix.exs`. I verified that it works with a packaged distillery release on my mac